dask/dataframe/core.py

```diff
@@ -3038,6 +3074,33 @@ def drop_columns(df, columns, dtype):
     return df
 
 
+def idxmaxmin_chunk(x, **kwargs):
+    fn = kwargs['fn']
+    axis = kwargs['axis']
```

Comment:
Can we put these keyword arguments directly in the signature instead of having `**kwargs`?